Assets/Scripts/SS3D/Systems/Tile/TileSystem.cs

```diff
@@ -15,8 +17,12 @@ namespace SS3D.Systems.Tile
     /// </summary>
     public class TileSystem : NetworkSystem
     {
-        public TileResourceLoader Loader { get; private set; }
+	    public const string SavePath = "/Tilemaps";
+
+	    public const string UnnamedMapName = "UnnamedMap";
```

Comment:
why do we have hardcoded strings there ? What's the unnamedMap ?

Reply:
Hardcoded strings are find as long as they are stored in a const or static variable. Its the default name for a map that has been created and not named by the user.